helix_fhir_client_sdk/fhir_client.py

```diff
@@ -1620,6 +1623,21 @@ async def refresh_token(
 
         return refresh_token
 
+    @staticmethod
+    def get_variables_to_log(vars_dict: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Method to return the variables which we need to log
+        :param vars_dict: (dict) dictionary of variables names with their values
+        """
+        variables_to_log = {}
+        for key, value in vars_dict.items():
+            if not isinstance(value, type(threading.Lock)):
+                variables_to_log[key] = value
+        variables_to_log.pop("_access_token")
+        variables_to_log.pop("_login_token")
+        variables_to_log.pop("_client_id")
+        return variables_to_log
```

Comment:
Are we getting only 3 types of secret logs in every case?

Reply:
yes